src/tests/functional/shared_test_classes/include/shared_test_classes/base/layer_test_utils.hpp

```diff
@@ -23,128 +21,9 @@
-                        const std::vector<InferenceEngine::Blob::Ptr> &actual,
-                        float threshold,
-                        float abs_threshold = -1.f);
-
-    static void Compare(const std::pair<ov::element::Type, std::vector<std::uint8_t>> &expected,
-                        const InferenceEngine::Blob::Ptr &actual,
-                        float threshold,
-                        float abs_threshold = -1.f);
-
-    virtual void Compare(const std::vector<std::pair<ov::element::Type, std::vector<std::uint8_t>>> &expectedOutputs,
-                         const std::vector<InferenceEngine::Blob::Ptr> &actualOutputs);
-
-    virtual void Compare(const std::pair<ov::element::Type, std::vector<std::uint8_t>> &expected, const InferenceEngine::Blob::Ptr &actual);
-
-    virtual void Compare(const InferenceEngine::Blob::Ptr &expected, const InferenceEngine::Blob::Ptr &actual);
-
-    virtual void Compare(const InferenceEngine::TensorDesc &actualDesc, const InferenceEngine::TensorDesc &expectedDesc);
-
-    std::shared_ptr<ov::Model> GetFunction();
-
-    std::map<std::string, std::string>& GetConfiguration();
-
-    template<class T_IE, class T_NGRAPH>
-    static void Compare(const T_NGRAPH *expected, const T_IE *actual, std::size_t size, float threshold, float abs_threshold = -1.f) {
-        for (std::size_t i = 0; i < size; ++i) {
-            const T_NGRAPH &ref = expected[i];
-            const auto &res = actual[i];
-            const auto absoluteDifference = ov::test::utils::ie_abs(res - ref);
-            if (abs_threshold > 0.f && absoluteDifference > abs_threshold) {
-                IE_THROW() << "Absolute comparison of values expected: " << std::to_string(ref) << " and actual: " << std::to_string(res)
-                           << " at index " << i << " with absolute threshold " << abs_threshold
-                           << " failed";
-            }
-            if (absoluteDifference <= threshold) {
-                continue;
-            }
-            double max;
-            if (sizeof(T_IE) < sizeof(T_NGRAPH)) {
-                max = static_cast<double>(std::max(ov::test::utils::ie_abs(T_NGRAPH(res)), ov::test::utils::ie_abs(ref)));
-            } else {
-                max = static_cast<double>(std::max(ov::test::utils::ie_abs(res), ov::test::utils::ie_abs(T_IE(ref))));
-            }
-            double diff = static_cast<float>(absoluteDifference) / max;
-            if (max == 0 || (diff > static_cast<float>(threshold)) ||
-                (std::isnan(static_cast<float>(res)) ^ std::isnan(static_cast<float>(ref)))) {
-                IE_THROW() << "Relative comparison of values expected: " << std::to_string(ref) << " and actual: " << std::to_string(res)
-                           << " at index " << i << " with threshold " << threshold
-                           << " failed";
-            }
-        }
-    }
-
+class LayerTestsCommon {
```

Comment:
File cannot be fully remove because:
1. Some `CoreConfiguration()` still references to `LayerTestsCommon`
See https://github.com/intel-innersource/applications.ai.vpu-accelerators.vpux-plugin/pull/8794 and https://github.com/openvinotoolkit/openvino_contrib/pull/864
2. Need a lot of work to replace headers from the file to places where it's required (TBD in next PRs)